docs/snippets/all/archetypes/transform3d_partial_updates.cpp

```diff
@@ -22,36 +22,35 @@ int main() {
     // Update only the rotation of the box.
     for (int deg = 0; deg <= 45; deg++) {
         auto rad = truncated_radians(deg * 4);
-        // TODO(#8583): update_fields
         rec.log(
             "box",
-            rerun::Transform3D().with_rotation_axis_angle(
+            rerun::Transform3D::update_fields().with_rotation_axis_angle(
                 rerun::RotationAxisAngle({0.0f, 1.0f, 0.0f}, rerun::Angle::radians(rad))
             )
         );
     }
 
     // Update only the position of the box.
-    for (int t = 0; t <= 45; t++) {
+    for (int t = 0; t <= 50; t++) {
```

Comment:
I'm not sure why this changed, but yeah this has to be `45` for the roundtrip to work.
```cpp
    for (int t = 0; t <= 45; t++) {

Reply:
No we have 50 box updates in all 3 tests, but 45 degree updates. This isn't the roundtrip that fails, `transform` roundtrip fails instead, apparently I broke something in the _existing_ api